app/models/penaltyDetails/appealInfo/AppealInformationType.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.penaltyDetails.appealInfo
+
+import play.api.libs.json.{JsValue, Json, Reads, Writes}
+import models.penaltyDetails.appealInfo.AppealLevelEnum
+
+case class AppealInformationType(
+                                  appealStatus: Option[AppealStatusEnum.Value],
+                                  appealLevel: Option[AppealLevelEnum.Value],
+                                  appealDescription: Option[String] //NOTE: this field is required in 1812 spec but has been set to optional as it is only used by 3rd party APIs
```

Comment:
Will we add a layer of validation in the Service (or other) layer where if the regime is VATC we check that optional fields are filled and arrays only have one element?